maps-app/src/main/java/com/esri/android/mapsapp/MapFragment.java

```diff
@@ -79,6 +79,8 @@
 import com.esri.android.mapsapp.tools.Compass;
 import com.esri.android.mapsapp.util.TaskExecutor;
 import com.esri.arcgisruntime.ArcGISRuntimeEnvironment;
+import com.esri.arcgisruntime.LicenseLevel;
+import com.esri.arcgisruntime.LicenseResult;
```

Comment:
The `ArcGISRuntimeEnvironment`, `LicenseLevel`, & `LicenseResult` imports are unused, please remove.